app/jobs/stats_for_recent_comments_job.rb

```diff
@@ -0,0 +1,45 @@
+class StatsForRecentCommentsJob < ActiveJob::Base
```

Comment:
Much like Rake tasks, I enjoy having background jobs have almost no logic inside them; instead, I like moving that logic to another class which the background job runs.

Advantages:

* The logic can be better-structured since the second, non-background-job class can have a real `initialize` method
* Easier to test a plain Ruby class

In this case, how about moving almost all of the `private` methods to another class, perhaps named `StatsCalculator`?

Reply:
Good idea. Done in caa6f26